src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -237,6 +237,8 @@ internal void RefreshLibraryView(WebView2 browser)
         /// <param name="text">text to be added to clipboard</param>
         internal void OnCopyToClipboard(string text)
         {
+            dynamoViewModel.Model.ClipBoard.Clear();
+            Clipboard.Clear();
```

Comment:
Why do we need to clear both clipboard?

Reply:
I forgot to remove that statement, I've just update the PR.
I'm going to set the PR as a draft as I realized there's an use case that need to be addressed before wrap this up.

Reply:
@Enzo707 this is the approach that I was talking about when implemeting the functionality of copying an external text to Dynamo, then you need to do the next steps to reproduce the bug:

1. Add a Number node in workspace
2. Select the Number node and press Ctrl + C
3. Then press Ctrl + V (the node will be added/copied in workspace)
4. The go to an external app (like notepad), select a text and press Ctrl + C
5. Return to Dynamo, go to the Search TextBox in Library and press Ctrl + V
**The problem is that is copying the text from Notepad to the Search TextBox in Library and also adding/copying a Number node in workspace.**

Then for fixing this problem my approach was to add a variable that holds the focused element, in this case the focused element will be the SearchTextBox, the missing part (TODO) in my code is that once you select a node or another item in the workspace you need to set the focused element variable to DynamoWorkspace or something like that so in that way we enable the copy/paste again in DynamoView.xaml.cs.

This is the line used for subcribing to the focus event (I executed it using the DeveloperTools), probably needs to be added in React when the page is already loaded because when I added it to library.html was crashing the page.

`document.getElementsByClassName("SearchInputText")[0].addEventListener("focus", focusedSearchTextBox);`

This is a GIF showing the behavior.
![devenv_jvpBVspJCY](https://github.com/DynamoDS/Dynamo/assets/61755417/813b33d2-b529-4c21-98ec-d23eea90d8c5)

This is the branch in which I uploaded the code:
https://github.com/RobertGlobant20/Dynamo/tree/DYN-6361-CopyPaste-FixProposal